apps/src/templates/CopyrightInfo.jsx

```diff
@@ -5,7 +5,10 @@ import i18n from '@cdo/locale';
 const CopyrightInfo = () => (
   <div>
     <a href="https://creativecommons.org/">
-      <img src="https://curriculum.code.org/static/img/creativeCommons-by-nc-sa.png" />
+      <img
+        src="https://curriculum.code.org/static/img/creativeCommons-by-nc-sa.png"
```

Comment:
Since we're deprecating this URL, this asset will need to be moved, right? If so, I'll create a ticket for it

Reply:
Great –– tracked here https://codedotorg.atlassian.net/browse/PLAT-1956, and mentioned in follow-up work